packages/lib/src/ui/blocks/PersonList/personListDef.tsx

```diff
@@ -0,0 +1,95 @@
+import * as yup from "yup"
+import { BlockTemplates, InputType } from "../enums"
+import { Block, BlockDef } from "../blocks"
+import { BlockFields, withBlockSchema } from "../../components/Block/Block"
+import { PersonProps } from "./PersonList"
+
+export interface PersonListBlock extends Block {
```

Comment:
Anglicky moc neumím, ale nemělo by to být `PersonsList`?

Reply:
Spíše ne. List of objects => object list. Ještě by mohlo být people, ale to mi přijde lepší uvést, že je to seznam.